liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/MavenUtils.java

```diff
@@ -100,7 +100,8 @@ public static boolean checkProLicense(String liquibaseProLicenseKey, String comm
         if (liquibaseProLicenseKey == null) {
             log.info("");
             if (commandName != null) {
-                log.info("The command '" + commandName + "' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com.");
+                log.info("The command '" + commandName + "' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com." +
```

Comment:
This code is just for maven, so the --pro-license-key part doesn't apply here. There is a different way we should be explaining how to set it up in maven